packages/core/src/components/forms/numericInput.tsx

```diff
@@ -406,10 +406,12 @@ export class NumericInput extends AbstractPureComponent2<HTMLInputProps & INumer
             // keydown is fired repeatedly when held so it's implicitly continuous
             onKeyDown: evt => {
                 if (Keys.isKeyboardClick(evt.keyCode)) {
+                    if (this.props.disabled) return;
```

Comment:
I am surprised we don't have a lint rule for this... I swear we used to... please use `{ }` braces for all if statements.

Reply:
also I think this check should go outside the `if (Keys.isKeyboardClick)` statement... or just be merged into the same conditional